packages/graphql-typescript-definitions/src/cli.ts

```diff
@@ -47,31 +54,62 @@ const argv = yargs
   })
   .help().argv;
 
+if (typeof argv.cwd === 'string') {
+  process.chdir(argv.cwd);
+}
```

Comment:
I think this at least needs some comments to explain, IMO it is generally not good practice to do this

Reply:
if you're not on board with it, i can go back to processing paths based on this value (and default to `process.cwd()`).